### PR TITLE
feat: add meterial components

### PR DESCRIPTION
**Pull request by mhevery**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -6,5 +6,6 @@
     "**/.DS_Store": true,
     "dist": true,
     "tmp": true
-  }
+  },
+  "typescript.tsdk": "/Users/misko/work/periscope/node_modules/typescript/lib"
```
Comment (IgorMinar) [inline .vscode/settings.json]: this should not be checked in

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (vicb) [inline .vscode/launch.json]: please `.gitignore` this

**Comment by vicb** [comment]
LGTM after `.vscode` config is removed

File: .vscode/launch.json (hunk 2 above)
Comment (mhevery) [inline .vscode/launch.json]: done

File: .vscode/settings.json (hunk 1 above)
Comment (mhevery) [inline .vscode/settings.json]: done

**Pull request merged**